src/chains.jl

```diff
@@ -86,6 +86,15 @@ Chains(chain::Chains, ::Nothing) = chain
 
 # Groups of parameters
 
+"""
+    namesingroup(chains::Chains, sym::Union{String,Symbol})
+
+Returns the names of all parameters in a chain that belong to the group `:name`.
```

Comment:
Docstrings should start with a single one-line sentence that describes the function in the imperative form (see https://docs.julialang.org/en/v1/manual/documentation/):
```suggestion
Return the names of all parameters in a chain that belong to the group `:name`.

```